src/resources/value-converters/dateDiff.ts

```diff
@@ -0,0 +1,20 @@
+﻿import { autoinject } from "aurelia-framework";
+import { DateService } from "../../services/DateService";
+
+@autoinject
+export class DateDiffValueConverter {
```

Comment:
We could also utilize "signalable value converters"
https://aurelia.io/docs/binding/value-converters#signalable-value-converters=

as in `app.ts` we already use something like that
`this.signaler.signal("secondPassed");`

Would have to update the code a bit though.
Could be a nice future improvement